src/main/java/com/bp3x/raidbot/commands/lfg/DeleteCommand.java

```diff
@@ -57,8 +60,22 @@ protected void execute(CommandEvent commandEvent) {
            MessageUtils.sendAutoDeletedMessage("Invalid arguments. Use `!delete <eventID>`", 300, commandEvent);
         }
         if (eventToDelete != null && messageToDelete != null) {
-            messageToDelete.delete().queue();
-            MessageUtils.sendAutoDeletedMessage("Deleted event with ID: " + eventToDelete.getEventId(), 300, commandEvent);
+            Message latestMessageInstance = commandEvent.getChannel().retrieveMessageById(messageToDelete.getId()).complete();
+            if (latestMessageInstance != null) {
+                ThreadChannel thread = latestMessageInstance.getStartedThread();
+                if (thread != null) {
+                    thread.delete().complete();
```

Comment:
I think this can be changed to .queue() as well. You can use the Consumer params to send the success/failure messages.